Sources/PathKit.swift

```diff
@@ -152,8 +152,15 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], self.path.contains(home) else { return self }
```

Comment:
I think it would be a good idea to make sure that `HOME` is actually set. Otherwise when abbreviating paths when home is unset (empty string) it may result in any path being abbreviated with `~`.